jerry-core/parser/regexp/re-parser.cpp

```diff
@@ -71,6 +71,27 @@ re_parse_non_greedy_char (lit_utf8_iterator_t *iter_p) /**< RegExp pattern */
 } /* re_parse_non_greedy_char */
 
 /**
+ * Parse a max 3 digit long octal number from input string iterator.
+ *
+ * @return uint32_t - parsed octal number
+ */
+static uint32_t
+re_parse_octal (lit_utf8_iterator_t *iter) /**< input string iterator */
+{
+  uint32_t number = 0;
+  for (int index = 0;
+       index < 3
+       && !lit_utf8_iterator_is_eos (iter)
+       && lit_char_is_octal_digit (lit_utf8_iterator_peek_next (iter));
+       index++)
+  {
+    number = number * 8 + lit_char_hex_to_int (lit_utf8_iterator_read_next (iter));
```

Comment:
hex_to_int? I suspect there is no octal, and this decrease code size. Ok, keep it.